index.html

```diff
@@ -5931,7 +5931,31 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content represents additional information or parenthetical context to the primary content it supplements. A <code>note</code> is content provided by the author of the page or document, it is not to be used for providing reactions or suggestions. For these purposes, please review <rref>comment</rref> and <rref>suggestion</rref>.</p>
+				<p>When used within the normal flow of a page's content, a <code>note</code> has an implicit association with the content that it supplements. The following example demonstrates using a <code>note</code> to call out additional information in the natural reading order of a page:</p>
+				<pre class="example highlight">
+					&lt;p>... the following results outline support for the tested features.&lt;/p>
+					&lt;div>
```

Comment:
should this have role="note"? If not I'm not sure what this example is doing here.

Reply:
see?  told you i needed a reviewer :)